pkg/operation/botanist/addons.go

```
@@ -311,6 +318,20 @@ func (b *Botanist) generateCoreAddonsChart(ctx context.Context) (*chartrenderer.
 	}
 	shootInfo["extensions"] = strings.Join(extensions, ",")
 
+	coreDNSRestartTimestamp, err := b.getCoreDNSRestartTimestamp(ctx)
+	if err != nil {
+		return nil, err
```

Should `not found` errors be ignored here?